addons/storyshots/stories/required_with_context/Button.stories.js

```diff
@@ -3,7 +3,7 @@ import React from 'react';
 import { storiesOf } from '@storybook/react'; // eslint-disable-line
 import { action } from '@storybook/addon-actions'; // eslint-disable-line
 
-import Button from './Button';
+import Button from '@storybook/components/dist/demo/Button';
```

Comment:
@tmeasday I'll think about it. Exposing all components as 1 module would mean we should allow tree shaking, because otherwise there's the potential for package bloat.

In fact maybe we should open an issue for exposing the es-modules code for all packages https://github.com/rollup/rollup/wiki/pkg.module